plugins/poll/assets/javascripts/components/poll-breakdown-option.js.es6

```diff
@@ -0,0 +1,56 @@
+import I18n from "I18n";
+import Component from "@ember/component";
+import { action } from "@ember/object";
+import { htmlSafe } from "@ember/template";
+import discourseComputed from "discourse-common/utils/decorators";
+import { getColors } from "../lib/chart-colors";
```

Comment:
I prefer absolute paths in our app rather than relative because I find it makes copying/pasting and searching easier